libraries/radspreadprocessing/features/hyperlink.md

```diff
@@ -89,12 +99,12 @@ __Example 3__ create a hyperlink to an email address.
 To add a hyperlink, you need to specify a cell range that will contain the hyperlink and a hyperlink info that will determine the type of the hyperlink.
         
 
-__Example 4__ assigns the hyperlink created in __Example 1__ to A1.
+__Example 5__ assigns the hyperlink created in __Example 1__ to A1.
         
 
-#### __[C#] Example 4: Add hyperlink__
+#### __[C#] Example 5: Add hyperlink__
 
-{{region cs-radspreadprocessing-features-hyperlink_3}}
```

Comment:
Changing the name of the region is not necessary. If you decide to leave them modified, make sure to also update the region names in the TFS project with the snippets as well. Also make sure that any new snippet is included in TFS

Reply:
I've not done this myself.